package.json

```diff
@@ -12,18 +12,25 @@
     "typescript",
     "openapi-client",
     "openapi-generator",
-    "@onfido/api"
+    "@onfido/api",
+    "onfido",
+    "identity",
+    "verification",
+    "api"
   ],
-  "license": "MIT",         
+  "license": "MIT",
   "main": "./dist/index.js",
   "typings": "./dist/index.d.ts",
   "module": "./dist/esm/index.js",
   "sideEffects": false,
   "scripts": {
-    "test": "jest", 
+    "test": "jest",
     "build": "tsc && tsc -p tsconfig.esm.json",
     "prepare": "npm run build"
   },
+  "files": [
+    "/dist"
+  ],
```

Comment:
This is to obtain same package structure in npm archive as [old npm package (v2.9.0)](https://www.npmjs.com/package/@onfido/api/v/2.9.0?activeTab=code).

[Example](https://github.com/onfido/onfido-node/actions/runs/8969224299/job/24630303554) of a npm package session (with `--dry-run` option):
```sh
npm notice 
npm notice 📦  @onfido/api@3.0.0
npm notice === Tarball Contents === 
npm notice 1.1kB   LICENSE                             
npm notice 5.4kB   README.md                           
npm notice 373.[8](https://github.com/onfido/onfido-node/actions/runs/8969224299/job/24630303554#step:5:9)kB dist/api.d.ts                       
npm notice 2[9](https://github.com/onfido/onfido-node/actions/runs/8969224299/job/24630303554#step:5:10)2.1kB dist/api.js                         
npm notice 1.4kB   dist/base.d.ts                      
npm notice 1.5kB   dist/base.js                        
npm notice 1.8kB   dist/common.d.ts                    
npm notice 7.2kB   dist/common.js                      
npm notice 2.5kB   dist/configuration.d.ts             
npm notice 2.2kB   dist/configuration.js               
npm notice 373.8kB dist/esm/api.d.ts                   
npm notice 283.0kB dist/esm/api.js                     
npm notice 1.4kB   dist/esm/base.d.ts                  
npm notice 1.2kB   dist/esm/base.js                    
npm notice 1.8kB   dist/esm/common.d.ts                
npm notice 6.4kB   dist/esm/common.js                  
npm notice 2.5kB   dist/esm/configuration.d.ts         
npm notice 2.0kB   dist/esm/configuration.js           
npm notice 366B    dist/esm/index.d.ts                 
npm notice 408B    dist/esm/index.js                   
npm notice 340B    dist/esm/webhook-event-verifier.d.ts
npm notice 1.2kB   dist/esm/webhook-event-verifier.js  
npm notice 366B    dist/index.d.ts                     
npm notice 1.2kB   dist/index.js                       
npm notice 340B    dist/webhook-event-verifier.d.ts    
npm notice 1.4kB   dist/webhook-event-verifier.js      
npm notice 1.4kB   package.json                        
npm notice === Tarball Details === 
npm notice name:          @onfido/api                             
npm notice version:       3.0.0                                   
npm notice filename:      @onfido/api-3.0.0.tgz                   
npm notice package size:  [12](https://github.com/onfido/onfido-node/actions/runs/8969224299/job/24630303554#step:5:13)2.5 kB                                
npm notice unpacked size: 1.4 MB                                  
npm notice shasum:        01d87704ed4dbf4387d6ac83776bf49eb5f74907
npm notice integrity:     sha512-03gzgmw0lJ9MX[...]krm/h7EIAueSQ==
npm notice total files:   27                                      
npm notice 
npm WARN This command requires you to be logged in to github:onfido/onfido-node (dry-run)
npm notice Publishing to github:onfido/onfido-node (dry-run)
+ @onfido/api@3.0.0
```
Change will be persisted via https://github.com/onfido/onfido-openapi-spec/pull/71.